scripts/e2e-tests

```diff
@@ -5,6 +5,24 @@
 # Builds the wasm locally, the e2e base image, moves the wasm to the
 # appropriate location, and runs the tests
 
+name=''
+
+help() {
+  echo "Usage:"
+  echo "    To run all tests:       ./scripts/e2e-tests"
+  echo "    To run a singular test: ./scripts/e2e-tests -n test_name_here"
```

Comment:
```suggestion
  echo "    To run a singular test: ./scripts/e2e-tests -n test_name"
```

Reply:
There are different notations for parameters (TEST_NAME, [test name],  ...). I don't have a strong preference. We should simply use consistent notation.

Reply:
Decided to go with `TEST_NAME`.